### ISSUE TITLE
Documentation: Should we add links to new community videos on migration topics

### ISSUE THREAD

**Issue by ysuarez** [Islandora/documentation#1579]
The community has created several videos related to migrations in the last several months, and I would like look into updating the following Islandora 8 documentation page with new video content.

The page is called: 

Islandora 7
https://islandora.github.io/documentation/technical-documentation/migrate-7x/

Currently this page above contains this video...

1) Migrating from Islandora 7.x to Islandora CLAW
https://www.youtube.com/watch?v=fEDzfSPjKEo
Jan 28, 2019

At this point I have watched the following Islandora 8 migration related videos in the foundation Youtube channel... 

2) Islandora Webinar: Migrating from Islandora 7 to Islandora 8
https://www.youtube.com/watch?v=n0kLOB8ZdbM
Nov 21, 2019

3) Islandora Online: Islandora Migration Tools
https://www.youtube.com/watch?v=95Bnix-z1zY&feature=em-uploademail
Aug 10, 2020

4) Islandora Online: CONTENTdm to Islandora 8; Remediation & Migration
https://www.youtube.com/watch?v=rON8-Dq_lY8&feature=em-uploademail
Aug 10, 2020

Here are the questions I have so far...

A) Should the currently listed video # 1 from above be kept or is it better for it to be removed, for example because the material is out of date?

B) Should any or all of the videos # 2- # 4 from above be added to this page? 

Thoughts? 

BTW, I already started communicating with @manez to work together on linking these and other new community videos on the doc site. Thanks.

**Comment by ysuarez** [comment]
@dannylamb thoughts on the currently listed video... 

Migrating from Islandora 7.x to Islandora CLAW
https://www.youtube.com/watch?v=fEDzfSPjKEo
Jan 28, 2019

Should it be kept or replaced by one of the newer ones?

Thanks

**Comment by manez** [comment]
@ysuarez I think the November webinar can replace that older video. The tool hasn't changed _a lot_, but we might as well stick with one demo since they content overlaps.

**Comment by dannylamb** [comment]
I agree with @manez.  Please take the newer one @ysuarez

**Comment by ysuarez** [comment]
Thank you both @dannylamb & @manez , I have a couple more questions

1) I am going to replace the current and older video with the one from November 2019

2) should I add any of the other videos I suggested above on this page we are discussing...

https://islandora.github.io/documentation/technical-documentation/migrate-7x/

Alternatively, I can create a new page called something like "migration approaches" "or migration overview"  at the same page level as the /migrate-7x/ page that links to this video...

Islandora Online: Islandora Migration Tools
https://www.youtube.com/watch?v=95Bnix-z1zY&feature=em-uploademail
Aug 10, 2020

I can then write a very short summary of how in this video you list the currently available approaches to migration? 

TIA

**Comment by manez** [comment]
@ysuarez I think I'm inclined towards the new page? If only because I suspect that whole section is going to have to grow a lot in the coming months, so we might was well start building it out now.

**Comment by dannylamb** [comment]
Resolved via https://github.com/Islandora/documentation/commit/a4bf86b03af62bb6cf09c4e840157516186ac004

**Comment by dannylamb** [comment]
Gotta say, the end result looks great!  Thanks everyone :smile:

**Comment by ysuarez** [comment]
I wanted to make a clarification here and a quick question. 

- The content is of my commits are like 99% taken from the text from Danny's slides. Hence some of the phrasing may feel off. I just copied it over verbatim so we would have something to build on. 

- I want to make some small changes. Should I create a new issue or open this one up?

For example, I thought I saw that several of the bullet list indentations did not display properly. They seem to all show up on the leftmost level throughout the document, not what my previous showed when editing. Also, @whikloj had a couple of good suggestions I wanted to work on. 

Thanks in advance, Yamil
